protobuf-codegen/src/gen/mod_rs.rs

```diff
@@ -3,9 +3,14 @@ use crate::gen::code_writer::CodeWriter;
 
 pub(crate) fn gen_mod_rs(mods: &[String]) -> compiler_plugin::GenResult {
     let v = CodeWriter::with_no_error(|w| {
-        w.comment(&format!("{}generated", "@"));
-        w.write_line("");
+        w.comment(&format!(
+            "This file is generated by {} {}. Do not edit",
+            env!("CARGO_PKG_NAME"),
+            env!("CARGO_PKG_VERSION")
+        ));
+        w.comment("@generated");
```

Comment:
That was

```
        w.comment(&format!("{}generated", "@"));
```

intentionally, so the generator is not considered to be generated.

Reply:
Reverted